sphinx/source/recovery.rst

```diff
@@ -1,105 +1,122 @@
 Catastrophic Recovery
 =====================
 
-For unexpected reasons, all nodes in the network may crash while executing business transactions. In this catastrophic scenario, it is possible that the length of ledgers of each node may differ slightly since some transactions may not have yet been fully replicated.
+For unexpected reasons, a significant number [#crash]_ of CCF nodes may crash. In this catastrophic scenario, it is possible that the length of ledgers of each node may differ slightly since some transactions may not have yet been fully replicated.
 
 .. note:: The current version of the recovery protocol relies on Intel SGX Sealing capability [#sealing]_.
 
 However, one of the previous ledgers can be recovered and the execution of new business transactions continue if the following three conditions are met:
 
 - At least one of the old nodes' CPU survived.
-- The sealed network secret file (``sealed_secrets.<date>.<pid>``) associated with that CPU is available to the members.
+- The sealed network secret file (``sealed_secrets.<date>.<pid>``) associated with that CPU is available to a quorum of members.
```

Comment:
We should probably define the quorum here as "enough members to pass a recovery vote according to the constitution contained in the recovery ledger".

Reply:
I have added an entry in the Glossary and a link to it wherever we mention "quorum" (as we do in quite a few places).